pytest_splunk_addon/splunk.py

```diff
@@ -549,13 +572,46 @@ def is_responsive_splunk(splunk):
             port=splunk["port"],
         )
         LOGGER.info("Connected to Splunk instance.")
+
         return True
     except Exception as e:
         LOGGER.warning(
-            "Could not connect to Splunk yet. Will try again. exception=%s", str(e),
+            "Could not connect to either Splunk or Splunk Forwarder yet. Will try again. exception=%s", str(e),
         )
         return False
 
+def is_responsive_hec(splunk, splunk_session, uri):
+    """
+    Verify if the hec port of Splunk is responsive or not
+
+    Args:
+        splunk (dict): details of the Splunk instance
+
+    Returns:
+        bool: True if Splunk HEC is responsive. False otherwise
+    """
+    try:
+        LOGGER.info(
+            "Trying to connect Splunk HEC...  splunk=%s", json.dumps(splunk),
+        )
+        response = requests.post(
+                "{}/{}".format(uri, "raw"),
+                auth=None,
+                data={"event":"test_is_responsive_hec"},
+                headers=splunk_session.headers,
+                verify=False,
+            )
+        LOGGER.debug("Status code: {}".format(response.status_code))
+        LOGGER.info("Splunk HEC is responsive.")
+        
+        return True
```

Comment:
Need to have a condition like this. 

https://github.com/splunk/pytest-splunk-addon/blob/0255220331ea12c78dbc463b42dd6c0d2beadb78/pytest_splunk_addon/splunk.py#L630-L632

Reply:
Done.